### PR TITLE
Add button

### PR DESCRIPTION
**Pull request by KevinDeLaTorre**
Redesigned the app a bit and added an Add button component, the button isn't actually functional but I got it to display correctly.

### CODE REVIEW

File: app/components/Events.vue [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
+                    <Label class="date textLeft" width="45%">D: {{ e.date }}</Label>
+                    <Label class="location textRight" width="55%">L: {{ e.location}}</Label>
+                </StackLayout>
+                <Label class="text time" width="100%">Start: {{ moment( e.start_time ) }}</Label>
+                <StackLayout class="text hbox " orientation="horizontal">
+                    <Label class="time textLeft" width="45%">End: {{ moment( e.end_time ) }}</Label>
+                    <Label class="people textRight" width="55%">P: {{ e.people }}</Label>
+                </StackLayout>
             </StackLayout>
+            <Label height="20px" />
```
Comment (lhwalsh) [inline app/components/Events.vue]: There must be a better way to do this than having a blank label lol. I'll leave another comment if I find anything to use instead.

File: app/components/Home.vue [hunk 2]
```diff
@@ -1,30 +1,36 @@
...
     <Page>
         <ActionBar>
-            <Label text="Pick Up Events" color="#18261F" />
+            <Label text="Pick Up Events" color="#ffffff" />
         </ActionBar>
-        <ScrollView>
-            <Events />
-        </ScrollView>
-    </Page>
+        <GridLayout rows="*" ref="rootLayout">
```
Comment (lhwalsh) [inline app/components/Home.vue]: Do we even need a `GridLayout`? Can we just have two tags not nested inside anything?

File: app/components/Home.vue [hunk 3]
```diff
@@ -1,30 +1,36 @@
...
 
 <script>
-    import Events from "./Events"
-    export default {
-        components: { Events }
-    }
+import Events from "./Events"
+import addButton from "./addButton"
+export default {
+components: { Events, addButton }
```
Comment (lhwalsh) [inline app/components/Home.vue]: you bitch, stop messing with the spacing

File: app/components/Home.vue [hunk 4]
```diff
@@ -1,30 +1,36 @@
...
+}
 </script>
 
 <style scoped lang="scss">
-    @import '~@nativescript/theme/scss/variables/purple';
+@import '~@nativescript/theme/scss/variables/purple';
```
Comment (lhwalsh) [inline app/components/Home.vue]: spacing 😫

**Review by lhwalsh — COMMENTED** [review]
Nice job, I'm interested to see how this looks in the app!

File: app/components/Events.vue (hunk 1 above)
Comment (KevinDeLaTorre) [reply app/components/Events.vue]: Industry standard.

File: app/components/Home.vue (hunk 4 above)
Comment (KevinDeLaTorre) [reply app/components/Home.vue]: I'mma look into vscode :(

**Pull request merged**